docs/environment-variables.md

```diff
@@ -35,6 +35,37 @@ Note that these environment variables may be removed at any time.
 | `WORKFLOW_GC_PERIOD` | `time.Duration` | `5m` | The periodicity for GC of workflows. |
 | `BUBBLE_ENTRY_TEMPLATE_ERR` | `bool` | `true` | Whether to bubble up template errors to workflow. |
 | `INFORMER_WRITE_BACK` | `bool` | `true` | Whether to write back to informer instead of catching up. |
+| `GRPC_MESSAGE_SIZE` | `string` | Use different GRPC Max message size for Argo server deployment (supporting huge workflows) |
+
+You can set environment variable for the argo-server deployment, for example:
```

Comment:
This is the section for the Controller, not the Server. It seems like a section for the Server did not exist at the time of this PR?

I moved it in #12148